File: src/Commands/Music/leave.ts
import { Command } from "../../Interfaces";

export const command: Command = {
  name: "leave",
  description: "Leaves the voice channel",
  category: "Music",
  aliases: ["l"],
  run: (variables) => {
    variables.client.player.leaveVC(variables);
  }
}

File: src/bot/commands/music/leave.rs
use crate::bot::commands::{Context, Error};

/// Leaves the voice channel
#[poise::command(
    slash_command,
    prefix_command,
    aliases("l"),
    category = "Music",
    guild_only
)]
pub async fn leave(ctx: Context<'_>) -> Result<(), Error> {
    let server = get_server!(ctx);

    server.player.leave_voice_channel(&ctx).await;
    message!(success, ctx, ("Left the voice channel"); true);

    Ok(())
}
